### ISSUE TITLE
Automatically set lang attribute on <article>

### ISSUE THREAD

**Issue by Minoru** [Minoru/blog.debiania.in.ua#40]
The front matter in my Markdown already has a language field. This should be used to set HTML `lang` attribute, which [helps screenreaders](https://blind.guru/htmllang.html).
